apps/examples/react-ssr-example/src/shared/components/atoms/Script/Script.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+interface ScriptProps {
+    async?: boolean;
+    defer?: boolean;
+    src?: string | string[];
```

Comment:
Same question, ever script needs a src right?

Reply:
Same answer as above. Probably should be defined but it makes it so that if the server can't find a script and passes null in the component will just not render.